tests/MoBi.Tests/Core/SBML/CompartmentImporterSpecs.cs

```diff
@@ -103,6 +105,12 @@ public void ContainerCreationTest3()
             containsSizeParam.ShouldBeTrue();
             containsVolumeParam.ShouldBeTrue();
          }
+
+         [Observation]
+         public void VolumeIsSetFromSize()
+         {
+            ((_container.Children.FirstOrDefault(x => x.Name == SBMLConstants.VOLUME) as IParameter).Formula as ConstantFormula).Value.ShouldBeEqualTo(5);
```

Comment:
Can we do sthg with less () and as
```
var parameter = _container.Parameter(SBMLConstants.VOLUME)
parameter.Value.ShouldBeEqualTo(5);
```
and if you want to check that the formula is a constant you can 
var formula = parameter.Formula as ConstantFormula (I prefer here to use Downcast<ConstantFormula> because it crashes if this is not the right type and as returns null
formula.Value etc..